src/cache.rs

```diff
@@ -142,41 +142,54 @@ impl Cache {
         }
     }
 
+    /// Check for pages for a given platform in one of the given languages.
+    fn find_page_for_platform(
+        &self,
+        name: &str,
+        cache_dir: &PathBuf,
```

Comment:
Passing in a `&PathBuf` is a bit strange, like passing in a `&String`. Instead, use a `&Path`.